### DISCUSSION TITLE
Your application cannot be reviewed until you fix the following

### DISCUSSION THREAD

**Discussion by XpPort** [community/community#42781] [GitHub Education]
I cannot get past the pre-check for the student developer pack. I have all the data that I can provide, but when checking a photo with an id, I get an error.

> Your application cannot be reviewed until you fix the following:
You are significantly more likely to be verified if you have completed your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) with your full name and a short bio.,Have you completed your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) with all your relevant information, such as your full name as it appears in your image and a short bio?

How can this problem be solved? Is it possible that for the citizens of Russia receiving is prohibited?

**Comment by SilasRodrigues19** [comment]
Hi @XpPort 

There are some discussions on this topic that have been resolved. Here are some links that may help you.

[https://github.com/community/community/discussions/38918](https://github.com/community/community/discussions/38918)
[https://github.com/community/community/discussions/30138](https://github.com/community/community/discussions/30138)
[https://github.com/community/community/discussions/37159](https://github.com/community/community/discussions/37159)

**Comment by hiten36** [comment]
It is not prohibited for anyone. Everyone has the opportunity to get this pack and its benefits. Try to edit your github profile - Name, bio, try adding social media links as well (it would help github to identify you), and every field. If this fails, add your government ID Card with your application and send it to github. Or try to mail github for the same.

**Reply by XpPort** [reply]
how can i send a letter? I mean to what mail and with what note and information
